rules/Flow_DetectWebcamByVendorMac.lua

```diff
@@ -0,0 +1,55 @@
+function Flow_DetectWebcamByVendorMAC (dpiMessage, ruleEngine)
+   -- Using this rule, Network Monitor can detect and alarm on the listed
+   -- internet enabled webcam devices by examining the MAC Address. 
+   -- The first three octets of a MAC address are defined according to IEEE 
+   -- standards and must be registered by the equipment manufacturer. A total 
+   -- of seven of the most common internet camera vendors are included in this script
+   --
+
+   require 'LOG'
+ 
+   if (TARGET_VENDOR_MAC_PREFIX == nil) then
+      -- List of Vendors MAC Address prefix we want to target
+      -- Note: Must be UPPER CASE for matching. More faster.
+      TARGET_VENDOR_MAC_PREFIX = {}
+      TARGET_VENDOR_MAC_PREFIX["30:8C:FB"] = true -- Dropcam aka. Nest Cam
+      TARGET_VENDOR_MAC_PREFIX["00:24:E4"] = true -- Withings
+      TARGET_VENDOR_MAC_PREFIX["00:18:03"] = true -- ArcSoft SimpliCam
+      TARGET_VENDOR_MAC_PREFIX["BC:51:FE"] = true -- Swann
+      TARGET_VENDOR_MAC_PREFIX["00:50:C2"] = true -- Y-Cam (1)
```

Comment:
This mac might not be valid. 
https://www.wireshark.org/tools/oui-lookup.html, and http://www.macvendors.com/ both claims it's IEEE registration (not yet registred).  Is this a typo?